### PR TITLE
Release 3.6.0

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.6.0 (2020-03-20)
+
+- Added the ability to delete rows of the grid. [#98](https://github.com/blackbaud/skyux-grids/pull/98)
```
Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md]: Do we want to specifically mention the inline delete component here? Or does that matter?

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Nope. We specifically hid the implementation details in the API

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Ah, understood.

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-release-3.6.0](https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-release-3.6.0)

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-DeniseCisneros) [reply CHANGELOG.md]: Looks like we specifically mention the grid component in other entries. 

Added the ability to delete rows in the grid component.

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Good call. Done

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-release-3.6.0](https://developer.blackbaud.com/skyux-grids?_sp.spa=pr-release-3.6.0)

**Pull request merged**